.circleci/config.yml

```diff
@@ -20,11 +27,11 @@ jobs:
                 key: dependency-cache-{{ checksum "package-lock.json" }}
                 paths:
                     - ./node_modules
-            # - run:
-            #     name: Test
-            #     command: |
-            #         mkdir -p allure-results
-            #         npm run test
+            - run:
+                name: Test
+                command: |
+                    mkdir -p allure-results
+                    npm run test:node
```

Comment:
when we create the new docker image, we'll just have to include a headless firefox install to do full test suite